### PR TITLE
HarmonyOS not HormonyOS

### PR DESCRIPTION
**Pull request by abossard**

### CODE REVIEW

**Comment by kimi-chen** [comment]
This mistake is a joke.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
-
 [English Docs](README.md)  |  [中文文档](doc/README-zh.md)
 
 
 # HarmonyOS
 ![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg) 
 
 <p align="center">
-  <a href="https://github.com/Awesome-HarmonyOS/HarmonyOS">
+  <a href="https://github.com/leisim/awesome-fuchsia">
```
Comment (colmugx) [inline README.md:8]: explain?

**Comment by hyan23** [comment]
百度翻译的别太较真

**Comment by abossard** [comment]
I think you already updated it in the master. I'm closing this PR.

File: README.md (hunk 1 above)
Comment (abossard) [reply README.md:8]: I don't know where this comes from. I just forked and replaces HormonyOS with HarmonyOS. Sorry.

Comment (abossard) [reply README.md:8]: I probably forked it very close to when it was fixed by @Awesome-HarmonyOS 
https://github.com/Awesome-HarmonyOS/HarmonyOS/commit/a3ba08a0712b8a4f79dd4dfc23d48992d6cdd568

**Comment by smile2014** [comment]
挂羊头卖狗肉

**Pull request closed without merging**